test_scripts/Defects/8_0/2845/common_2845.lua

```diff
@@ -0,0 +1,133 @@
+---------------------------------------------------------------------------------------------------
+-- Common module for tests of https://github.com/smartdevicelink/sdl_core/issues/2845 issue
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local rc = require('user_modules/sequences/remote_control')
+local utils = require('user_modules/utils')
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+--[[ Common Variables ]]
+local common = { }
+common.modules = rc.data.getRcModuleTypes()
+
+--[[ Common Functions ]]
+common.preconditions = actions.preconditions
+common.start = rc.rc.start
+common.registerAppWOPTU = actions.app.registerNoPTU
+common.defineRAMode = rc.rc.defineRAMode
+common.activateApp = actions.app.activate
+common.postconditions = actions.postconditions
+
+local function getRCAppConfig(pPt)
+  local moduleTypes = rc.data.getRcModuleTypes()
+  local groups = { "Base-4", "RemoteControl" }
+  local appHMIType = { "REMOTE_CONTROL" }
+  if pPt then
+    local out = utils.cloneTable(pPt.policy_table.app_policies.default)
+    out.moduleType = moduleTypes
+    out.groups = groups
+    out.AppHMIType = appHMIType
+    return out
+  else
+    return {
+      keep_context = false,
+      steal_focus = false,
+      priority = "NONE",
+      default_hmi = "NONE",
+      moduleType = moduleTypes,
+      groups = groups,
+      AppHMIType = appHMIType
+    }
+  end
+end
+
+function common.preparePreloadedPT(pRCAppIds)
+  local preloadedTable = actions.sdl.getPreloadedPT()
+  for _, rcAppId in pairs(pRCAppIds) do
+    local appId = config["application" .. rcAppId].registerAppInterfaceParams.fullAppID
```

Comment:
I propose to replace config["application" .. rcAppId].registerAppInterfaceParams by actions.app.getParams()